src/extension.ts

```diff
@@ -141,27 +133,52 @@ function getPRsPerSHAs() {
           "No search results. Try selecting a bigger piece of code or another file."
         );
       } else {
-        issuesBySHAs.forEach((issue: { url: any }) => {
-          const issueUrl = issue.url;
+        let issuesWithTitlesAndGroupedComments: { user: any; title: string; comments: any[]; created_at: any; }[] = [];
 
-          octokit
-            .request(`GET ${issueUrl}/comments`)
-            .then((octorespComments: any) => {
-              // this paints the panel
-              watermelonPanel.currentPanel?.doRefactor({
-                command: "prs",
-                data: octorespComments.data,
+        issuesBySHAs.forEach(async (issue: { url: any }) => {
+          const issueUrl = issue.url;
+          let prTitlesPushed: string[] = [];
+
+          await octokit.request(`GET ${issueUrl}/comments`).then(async (octoresp: { data: { issue_url: any; body: string; user: any; title: string; comments: any[]; created_at: any; }[]; }) => {
```

Comment:
Could we launch the `comment` request at the same time as the `title` one?
This would make them parallel